### PR TITLE
Add Audio8-ASR-0.1B H200 evaluation jobs

### PR DESCRIPTION
**Pull request by Alichua**
## Description

Adds [`AutoArk-AI/Audio8-ASR-0.1B`](https://huggingface.co/AutoArk-AI/Audio8-ASR-0.1B)
as an independent model family to the Open ASR Leaderboard.

Audio8-ASR-0.1B is a compact autoregressive multilingual ASR model with a
Qwen3-ASR audio encoder and an 8-layer Qwen-style causal language decoder. This
PR adds only `audio8_asr/submit_jobs.sh`; the existing `ark_asr` configuration
is intentionally unchanged.

Evaluation Space:
[`AutoArk-AI/open-asr-leaderboard-audio8-asr`](https://huggingface.co/spaces/AutoArk-AI/open-asr-leaderboard-audio8-asr)

Pinned model revision:
`b812eff124893ecd76a1dcde74ee58db5adab59c`

Run with:

```bash
RESULTS_BUCKET="AutoArk-AI/audio8-asr-open-asr-results" \
HF_TOKEN="hf_..." \
ORG_NAME="AutoArk-AI" \
bash audio8_asr/submit_jobs.sh
```

## Type of change

- [x] New model
- [ ] New dataset
- [ ] Bug fix
- [ ] New feature
- [ ] Other

## New Model Checklist

- [x] H200 WER results, seven-split mean WER, and composite RTFx are published
  in the model repository's
  [`.eval_results/open_asr_leaderboard.yaml`](https://huggingface.co/AutoArk-AI/Audio8-ASR-0.1B/blob/main/.eval_results/open_asr_leaderboard.yaml).
- [x] Dedicated Docker evaluation Space is public and reproducible.
- [x] `trust_remote_code` model loading is pinned to an immutable revision.
- [x] The same decoding hyperparameters are used across all seven datasets.
- [x] The runner saves raw, un-normalized model outputs.
- [x] The runner uses `normalizer/data_utils.py` for Hub loading, filtering, and
  manifest writing.
- [x] Mixed-length batch processing uses tokenizer left padding and explicit
  per-sample mel feature lengths.
- [x] Maximum selected stable H200 batch sizes are used per dataset.
- [x] Warmup and CUDA Event timing are enabled. `torch.compile` remains disabled
  because the pinned remote model code graph-breaks on dynamic KV-cache length.
- [x] End-to-end model parameter count is printed by the runner.
- [x] Raw H200 manifests and summaries are available in
  `hf://buckets/AutoArk-AI/audio8-asr-open-asr-results/AutoArk-AI-Audio8-ASR-0.1B`.

## Evaluation Settings

```text
dtype: bfloat16
attention: eager
decoding: greedy
max_new_tokens: 256
sampling_rate: 16000
max_audio_seconds: 30
prompt: Please transcribe this audio.
CUDA allocator: expandable_segments:True
```

| Split | Rows | Batch | WER | H200 RTFx |
| --- | ---: | ---: | ---: | ---: |
| AMI Cleaned | 7,715 | 1,152 | 10.99 | 396.91 |
| Earnings22 | 2,737 | 1,024 | 12.31 | 654.17 |
| GigaSpeech Cleaned | 18,757 | 1,408 | 8.48 | 641.19 |
| LibriSpeech test.clean | 2,620 | 1,024 | 2.70 | 687.84 |
| LibriSpeech test.other | 2,939 | 1,024 | 6.59 | 610.52 |
| SPGISpeech | 39,341 | 2,048 | 3.73 | 870.32 |
| VoxPopuli Cleaned AA | 628 | 628 | 4.39 | 686.14 |
| **Total / mean** | **74,737** | - | **7.03** | **741.15 composite** |

## Model Metadata

| License | Size (B) | # Languages | Encoder | Decoder |
| --- | ---: | ---: | --- | --- |
| Apache-2.0 | 0.324 | 7 | Qwen3-ASR audio encoder | 8-layer Qwen-style causal LM |

The parameter count is the full end-to-end unique count: `323,990,528`.

## Reproducibility Notes

- The final H200 bundle contains all 74,737 expected rows.
- Audio locators, durations, and normalized references match the independently
  validated local data identity multiset exactly.
- Per-split WER, insertion/deletion/substitution counts, RTFx, model revision,
  parameter count, and decoding settings were recomputed from the synced
  manifests.
- The first batch-1024 attempts exposed CUDA allocator fragmentation during the
  large logits projection. `PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True`
  resolved it without lowering the selected batch sizes.
- BF16 greedy outputs can differ slightly between singleton and batched decoder
  matrix shapes. The runner guarantees byte-identical valid mel prefixes and
  deterministic repetition of the same batch.
- Empty outputs and 256-token repetition loops are preserved without manual
  repair and remain included in WER.

### CODE REVIEW

File: audio8_asr/submit_jobs.sh [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+set -euo pipefail
+
+# Independent Audio8-ASR HF Jobs submission.
+# Usage:
+#   RESULTS_BUCKET="AutoArk-AI/audio8-asr-open-asr-results" \
+#   HF_TOKEN="hf_..." \
+#   ORG_NAME="AutoArk-AI" \
+#   bash audio8_asr/submit_jobs.sh
+
+SPACE="${SPACE:-AutoArk-AI/open-asr-leaderboard-audio8-asr}"
```
Comment (ebezzam) [inline audio8_asr/submit_jobs.sh]:
```suggestion
SPACE="${SPACE:-hf-audio/open-asr-leaderboard-audio8-asr}"
```

can you change to our copy?

File: audio8_asr/submit_jobs.sh [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+RESULTS_BUCKET="${RESULTS_BUCKET:?Set RESULTS_BUCKET to a writable Hugging Face bucket}"
```
Comment (ebezzam) [inline audio8_asr/submit_jobs.sh]:
```suggestion
RESULTS_BUCKET="${RESULTS_BUCKET:-hf-audio/asr_leaderboard_h200}"
```
can you default to our bucket? it will help for our ongoing maintainance/support

File: audio8_asr/submit_jobs.sh [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  echo "One or more HF Jobs failed" >&2
+  exit 1
+fi
+
+if [[ -n "$ORG_NAME" ]]; then
+  echo "Jobs page: https://huggingface.co/organizations/${ORG_NAME}/settings/jobs"
+else
+  echo "Jobs page: https://huggingface.co/settings/jobs"
+fi
+
```
Comment (ebezzam) [inline audio8_asr/submit_jobs.sh:118]: can you add a pause? I think we need time for bucket results to sync so that scoring can take place just after

```suggestion

wait
sleep 10  # allow time for the last results to be flushed to the bucket

```

File: audio8_asr/submit_jobs.sh [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+hf buckets sync \
+  "hf://buckets/${RESULTS_BUCKET}/${MODEL_FOLDER}" \
+  "$local_results"
+
+actual="$(find "$local_results" -maxdepth 1 -name '*.jsonl' | wc -l)"
+expected="${#DATASET_CONFIGS[@]}"
+if [[ "$actual" -ne "$expected" ]]; then
+  echo "Expected $expected JSONL manifests, found $actual in $local_results" >&2
+  exit 1
+fi
```
Comment (ebezzam) [inline audio8_asr/submit_jobs.sh]:
```suggestion
```
let's remove this. since you're writing 2x the number of files this exits and doesn't get to scoring below.

Also one may one to run a smoke test with one of the splits, and then do the rest.

**Review by ebezzam — COMMENTED** [review]
@Alichua thanks for the submission and for following our HF jobs guidelines so nicely!

There are the results I obtained (closely matching yours), can you confirm?

```
********************************************************************************
Results per dataset:
********************************************************************************
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 10.99 %, RTFx = 369.82
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_earnings22_test: WER = 12.31 %, RTFx = 629.68
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 8.48 %, RTFx = 613.50
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 2.70 %, RTFx = 680.01
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 6.59 %, RTFx = 588.84
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 3.73 %, RTFx = 832.74
AutoArk/AI-Audio8-ASR-0.1B | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 4.39 %, RTFx = 709.22

********************************************************************************
avg WER (AutoArk-AI/Audio8-ASR-0.1B) = 7.03
model,RTFx,License,Size (B),# Languages,Encoder,Decoder,AMI-Cleaned WER,Earnings22 WER,Gigaspeech-Cleaned WER,LS Clean WER,LS Other WER,SPGISpeech WER,Voxpopuli-Cleaned-AA WER
AutoArk-AI/Audio8-ASR-0.1B,709.29,,,,,,10.99,12.31,8.48,2.7,6.59,3.73,4.39
********************************************************************************
```

**Comment by Alichua** [comment]
Hi @ebezzam,  

Thanks for independently rerunning the evaluation and creating the hf-audio Space copy!  

All seven per-dataset WERs, as well as the mean WER of 7.03, match our results exactly. The composite RTFx is 709.29 compared with 741.15 in our run, and the per-split throughput values are reasonably close, so the reproduction looks solid.  

I’ve also updated the PR based on your feedback:
- set hf-audio/open-asr-leaderboard-audio8-asr as the default Space;
- set hf-audio/asr_leaderboard_h200 as the default results bucket;
- added a 10-second wait before syncing the bucket results;
- removed the strict manifest-count check to support partial and smoke runs.

Thanks again for taking the time to verify the results and help improve the integration!

**Comment by ebezzam** [comment]
Thanks for confirming! the results are live: https://huggingface.co/spaces/hf-audio/open_asr_leaderboard
